flycs_sdk/custom_code.py

```diff
@@ -90,19 +92,26 @@ def __init__(
         :param requirements: list of python package required by this code, use the same format as normal python requirements.txt files.
                              These package will be installed on the composer instance.
         :type requirements: List[str]
+        :param func_kwargs: List of kwargs areguments for a customer operator
+        :type func_kwargs: Dict[str:object]
+        :param run_before_keyset: override dependencies and only use the explicitly defined dependencies
```

Comment:
The docstring for this parameter is confusing. We always use the explicitly defined dependencies, in all cases. 
But here we do not talk about the keyset steps... while this is what this parameter is about.